### ISSUE TITLE
`// cSpell:disable` example

### ISSUE THREAD

**Issue by bubbletroubles** [streetsidesoftware/vscode-spell-checker#2948]
The example in the [readme.md](https://github.com/streetsidesoftware/vscode-spell-checker/blob/f09dab74de63eeabd9769542de67c6a5365d733b/README.md?plain=1#L207) file says that `// cSpell:disable`  syntex can be used.

However, that syntax is not an option in the [Disable checking](https://github.com/streetsidesoftware/vscode-spell-checker/blob/f09dab74de63eeabd9769542de67c6a5365d733b/README.md?plain=1#L188C1-L195C1) documentation a few lines above.

<img width="208" alt="image" src="https://github.com/streetsidesoftware/vscode-spell-checker/assets/42738824/65f30ce8-09d0-45cb-9421-70678e057eab">

It would be good to clarify the example vs valid ways to disable spell checking.

**Comment by Jason3S** [comment]
@bubbletroubles,

The important part is `cspell:disable`, not how to create a comment.

For example if you want to disable the spell checker in a Markdown file, you would use:

**Markdown**
```markdown
<!--- cspell:disable --->
This text is not checked.
<!--- cspell:enable --->
This text is checked.
```

**Python:**
```python
# cspell:disable
example = "This text is not checked."
# cspell:enable
example2 = "This text is checked."
```

Is this what you were asking about?

**Comment by bubbletroubles** [comment]
Ah yes that makes sense., thanks @Jason3S .

I created a draft PR #2949 to update the example (I'm not sure if you want to merge it, but it sure would have helped me to have that info). Let me know if I should publish the PR for your review.
